JSR_Shopping_List_Functions-master/js/app.js

```diff
@@ -0,0 +1,947 @@
+ (function() { // protect the lemmings!
+
+// 	/* 1
```

Comment:
I'm not quite sure why you commented out all your work, but when I uncommented several of them worked just fine! Just for future reference: you don't have to make a separate directory in the repository—your code will be submitted to your fork, so keeping the files in the same places is preferable.